### ISSUE TITLE
release 0.6.1 - No responses defined for platform : ACTIONS_ON_GOOGLE

### ISSUE THREAD

**Issue by KoffeeToKode** [dialogflow/dialogflow-fulfillment-nodejs#151]
Hello,
with the dialogflow-fulfillment release 0.6.1, we have the following problem :

When a followup event without any response message is set, we get the message
`No responses defined for platform : ACTIONS_ON_GOOGLE`

Thanks!

**Comment by matthewayne** [comment]
If you are still having this issue please open a new issue with the following information:
1. a description of the issue
2. your environment (version of OS/library/Node.js and/or server(less) platform)
3. code demonstrating the issue
4. the expected result
5. the actual result

**Comment by LukeJanissen** [comment]
Hi, I've got exactly the same issue. This happened after I updated the package. Is there a solution to this?

The expected result:
Switching to a new intent by using followup event

The result:
UnhandledPromiseRejectionWarning: Error: No responses defined for platform: ACTIONS_ON_GOOGLE

My code:
`const agent = new WebhookClient({request: request, response: response});`
`agent.context.set(shiftContext);` // shiftcontext is a valid context object
`agent.setFollowupEvent(eventName)`
